### PR TITLE
Add spanish strings

### PR DESCRIPTION
**Pull request by VMadalin**
### Description
Add support for the Spanish language as it was requested in https://github.com/googlesamples/android-sunflower/issues/462

### CODE REVIEW

File: app/src/main/res/values-es/strings.xml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+<resources>
+    <string name="menu_filter_by_grow_zone">Filtrar por zona de crecimiento</string>
+    <string name="plant_detail_image_content_description">Imagen de la planta</string>
+    <string name="my_garden_title">Mi jardín</string>
+    <string name="plant_list_title">Lista de plantas</string>
+    <string name="available">Plantas Disponibles</string>
+    <string name="plant_details_title">Detalles de la planta</string>
+    <string name="add_plant">Agregar planta</string>
+    <string name="added_plant_to_garden">Planta añadida al jardín</string>
+    <string name="garden_empty">Tu jardín esta vacio</string>
```
Comment (kristianconk) [inline app/src/main/res/values-es/strings.xml]: vacío

File: app/src/main/res/values-es/strings.xml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <string name="plant_date_header">Plantada</string>
+    <string name="watered_date_header">Ultima vez regada</string>
```
Comment (kristianconk) [inline app/src/main/res/values-es/strings.xml]: Última

File: app/src/main/res/values-es/strings.xml [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="menu_filter_by_grow_zone">Filtrar por zona de crecimiento</string>
+    <string name="plant_detail_image_content_description">Imagen de la planta</string>
+    <string name="my_garden_title">Mi jardín</string>
+    <string name="plant_list_title">Lista de plantas</string>
+    <string name="available">Plantas Disponibles</string>
```
Comment (manuelvicnt) [inline app/src/main/res/values-es/strings.xml]: Should "Disponibles" be lowercase?

File: app/src/main/res/values-es/strings.xml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    <string name="plant_details_title">Detalles de la planta</string>
+    <string name="add_plant">Agregar planta</string>
```
Comment (manuelvicnt) [inline app/src/main/res/values-es/strings.xml]: I'd prefer this to be "Añadir planta"

File: app/src/main/res/values-es/strings.xml [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    <string name="added_plant_to_garden">Planta añadida al jardín</string>
+    <string name="garden_empty">Tu jardín esta vacío</string>
+    <string name="plant_date_header">Plantada</string>
+    <string name="watered_date_header">Última vez regada</string>
+    <string name="menu_item_share_plant">Compartir</string>
+    <string name="share_text_plant">Revisa la planta %s en la aplicación "Sunflower"</string>
```
Comment (manuelvicnt) [inline app/src/main/res/values-es/strings.xml]: Would remove the quotes from Sunflower and would add Android in the description: "... en la aplicación Android Sunflower"

File: app/src/main/res/values-es/strings.xml [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+    <!-- String plurals and their related prefix / suffix strings -->
+    <string name="watering_needs_prefix">Necesita regarla</string>
```
Comment (manuelvicnt) [inline app/src/main/res/values-es/strings.xml]: Would prefer this to be "Necesita ser regada"

File: app/src/main/res/values-es/strings.xml [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+    <plurals name="watering_needs_suffix">
+        <item quantity="one">cada día.</item>
```
Comment (manuelvicnt) [inline app/src/main/res/values-es/strings.xml]: Would say "todos los días.". Feels more natural

Comment (VMadalin) [reply app/src/main/res/values-es/strings.xml]: Maybe we should change with `diariamente`. What do you think ?

**Comment by VMadalin** [comment]
Thanks to @manuelvicnt  and @kristianconk for your contribution helping this project to improve Spanish translations PR. I applied almost all of your suggestions pending to discuss one of them.

File: app/src/main/res/values-es/strings.xml (hunk 7 above)
Comment (manuelvicnt) [reply app/src/main/res/values-es/strings.xml]: Sounds good!

**Comment by manuelvicnt** [comment]
@tiembo this is ready to be merged :)

**Comment by tiembo** [comment]
Thank you @VMadalin for the PR and @manuelvicnt for the review!

Added Spanish string translation to the [Notable Community Contributions](https://github.com/android/sunflower/wiki/Notable-Community-Contributions) page

**Pull request merged**